LSE-63.tex

```diff
@@ -25,6 +25,8 @@
 \addtohist{5}{2011-07-10}{General edits}{Tony Tyson}
 \addtohist{6}{2011-07-15}{Added content}{Tony Tyson}
 \addtohist{7}{2011-07-31}{Added content}{Tony Tyson}
+\addtohist{8}{2017-04-13}{Added content}{Tim Jenness}
+\addtohist{7}{2017-04-16}{Added content}{Tony Tyson}
```

Comment:
I think the new LPM-51 policy is to only increment the release number at change control time. I can fix that up before merging.